dist/vespa.spec

```diff
@@ -232,7 +232,7 @@ Requires: vespa-protobuf = 3.7.0-5.el7
 %endif
 %if ! 0%{?amzn2}
 Requires: vespa-telegraf >= 1.1.1-1
-Requires: vespa-valgrind >= 3.16.0-1
+Requires: vespa-valgrind >= 3.17.0-1
```

Comment:
This package does not exist ?

Reply:
When will it be available ?

Reply:
.. After merge of https://github.com/vespa-engine/vespa-3rdparty-deps/pull/61

Reply:
This is currently a manual process. Sent you the details.